src/main/java/org/openmrs/isanteplus/performancedata/generator/predefined/UserEnum.java

```diff
@@ -0,0 +1,17 @@
+package org.openmrs.isanteplus.performancedata.generator.predefined;
+
+public enum UserEnum {
```

Comment:
Maybe it could be UserIdEnum, and the values would not require _ID then, as it's only ID's here.

Reply:
This class wouldn't be extendable in that case.